teos/src/dbm.rs

```diff
@@ -530,6 +634,66 @@ impl DBM {
         trackers
     }
 
+    /// Loads trackers with the given confirmation status.
+    ///
+    /// Note that for [`ConfirmationStatus::InMempoolSince(height)`] variant, this pulls trackers
+    /// with `h <= height` and not just `h = height`.
```

Comment:
Just a note to self for the final review. Do we need to keep track of this being on the mempool?

It only seems useful for reporting data with `get_appointment`

Reply:
Nvm, this is useful when pulling data for reorg purposes